docs/Tutorial_Top_Ten_3.ipynb

```diff
@@ -0,0 +1,251 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    T2 = T*(-1)
</pre><p>Please write it as:</p><pre class="ql-syntax" spellcheck="false">T_flipped = -1 * T
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/676/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>